client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/SubmissionEditStepForm.jsx

```diff
@@ -146,7 +146,11 @@ class SubmissionEditStepForm extends Component {
 
       let title = '';
       if (explanation.correct) {
-        title = intl.formatMessage(translations.correct);
+        if (question.type === questionTypes.TextResponse && !question.autoGradable) {
```

Comment:
I think this should be adopted to all non-autogradable questions (file upload, etc), what do you think?

Reply:
That sounds better. Let me improve this.